resources/README.md

```diff
@@ -17,5 +17,4 @@ Resources currently include, but are not limited to, the following:
 - Elements which are essential for the installation of `core` components in Kyma, such as certificates, users, and permissions
 - The `core` components required to run the Kyma
 - Examples of the use of specific components
-- The repository of bundles that are exposed in the Service Catalog as Service Classes through the Helm Broker
```

Comment:
please also adjust the documentation in this file:
- docs/service-brokers/docs/004-overview-helm-broker.md
- docs/service-brokers/docs/011-configuration-helm-broker.md
- docs/service-brokers/docs/021-architecture-helm-broker.md

Reply:
fixed